substrate/frame/transaction-payment/src/benchmarking.rs

```diff
@@ -38,6 +38,7 @@ fn assert_last_event<T: Config>(generic_event: <T as Config>::RuntimeEvent) {
 	T: Config,
 	T::RuntimeOrigin: AsTransactionAuthorizedOrigin,
 	T::RuntimeCall: Dispatchable<Info = DispatchInfo, PostInfo = PostDispatchInfo>,
+	<BalanceOf<T> as codec::HasCompact>::Type: DecodeWithMemTracking,
```

Comment:
same reasoning here.

Maybe I am wrong though, but is there anybody who would use the pallet without the benchmarks?

Reply:
> same reasoning here.

Answered above.

> Maybe I am wrong though, but is there anybody who would use the pallet without the benchmarks?

Not sure, but I guess not.